go/libraries/doltcore/env/actions/commit.go

```diff
@@ -40,6 +44,73 @@ type CommitStagedProps struct {
 	Email            string
 }
 
+// we are more permissive than what is documented.
+var SupportedLayouts = []string{
```

Comment:
actions is not a command line parsing package.  We shouldn't just throw things in here because its convenient and accessible in the other packages that you are working in.